### ISSUE TITLE
Support setMaxWidth for the TextView

### ISSUE THREAD

**Issue by CapnSpellcheck** [ViHtarb/Tooltip#24]
The TextView has a very handy setMaxWidth unlike normal views.
Expose a setMaxWidth in the Builder and as an attr as well, so one can control the popup window width.

Given enough time I will post a pull request

**Comment by CapnSpellcheck** [comment]
I'm too lazy to push a branch and do a PR at the moment :), but I can post a patch.
What I did was super simple, and you might even consider it incomplete since I didn't bother to support setMaxWidth via code. That's why there's a comment block, without any code :) but it would be easy to add a method.
I just used it with android:maxWidth in XML, worked great
[vihtarb-tooltip-issue-24.patch.zip](https://github.com/ViHtarb/Tooltip/files/880580/vihtarb-tooltip-issue-24.patch.zip)
